runtime/jcl/common/mgmtthread.c

```diff
@@ -1257,7 +1234,6 @@ initIDCache(JNIEnv *env)
 	if (JNI_ENOMEM == err) {
 		(*env)->ThrowNew(env, oom, "initIDCache failed");
 	}
```

Comment:
This is dead code: `err` is no longer set to `JNI_ENOMEM`.

Reply:
sorry, I didn't get that - there are 4 places where this is set - whenever the allocation routine fails. can you double check?

Reply:
Sorry, you're right, there are 3 places where it set to `JNI_ENOMEM`.